### PR TITLE
ES6 syntax addition

### PR DESCRIPTION
**Pull request by darkCavalier11**
Adding ES6 syntax of **const**, **let** and ES6 function syntax to node-native, node-request, node-unirest modules.

### CODE REVIEW

File: codegens/nodejs-native/lib/requestES6.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+  if (options.followRedirect) {
+    snippet = `const ${nativeModule} = require('follow-redirects').${nativeModule};\n`;
+  }
+  snippet += 'const fs = require(\'fs\');\n\n';
+  if (_.get(request, 'body.mode') && request.body.mode === 'urlencoded') {
+    snippet += 'const qs = require(\'querystring\');\n\n';
+  }
+
+  snippet += 'let options = {\n';
```
Comment (darkCavalier11) [inline codegens/nodejs-native/lib/requestES6.js]: let and const addition

**Comment by sastava007** [comment]
@darkCavalier11 
My PR is also failing for the same reason i.e. unfulfillment of travis requirements as mentioned in **`ci-requirements`**. Upon going through the log I found that there's no requirements mentioned in `ci-requirements.sh` file for any of the NodeJS codegen. But I can't understand why they haven't included requirements for all the codegens and just made it limited to few languages like Java/ C#/ Swift and pHp. 

What do you think?
**_Let's fix this together!_**

**Comment by darkCavalier11** [comment]
**sure, lets do this!**

**Comment by sastava007** [comment]
@darkCavalier11 
You changed the` travis.yml` file, any luck?

**Comment by shreys7** [comment]
Hey @darkCavalier11 @sastava007, We have identified the reason for constant failures on CI. All the extra dependencies are downloaded that are required to run snippets of various languages, as mentioned in the `ci-requirements.sh` file. Java dependencies are downloaded from central.maven.org to run java okhttp and java unirest tests, but it throws the following error:

`Resolving central.maven.org (central.maven.org)... failed: Name or service not known.
wget: unable to resolve host address ‘central.maven.org’`

We are fixing this. Will keep you updated 👍

**Comment by sastava007** [comment]
@shreys7 Thanks for letting us know.
Can you please restart my build again after the CI is fixed and running?

**Pull request closed without merging**